RubeesEat/wwwroot/js/SplitBill.js

```diff
@@ -5,9 +5,13 @@ function AddPerson() {
     const selectedPerson = selectElement.value;
     const selectedId = selectElement.selectedOptions[0].dataset["id"];
 
-    const personSpan = document.createElement("span");
-    personSpan.textContent = selectedPerson;
-    document.querySelector("#addedPeople").appendChild(personSpan);
+    const personDiv = document.createElement('div');
+    const textDiv = document.createElement('div');
+
+    personDiv.className = 'personDiv';
+    textDiv.className = 'text';
+    textDiv.textContent = selectedPerson + " ";
+    personDiv.appendChild(textDiv);
```

Comment:
I would suggest reordering the statements to fit with the variable declarations.

```suggestion
    const personDiv = document.createElement('div');
    personDiv.className = 'personDiv';

    const textDiv = document.createElement('div');
    textDiv.className = 'text';
    textDiv.textContent = selectedPerson + " ";
    personDiv.appendChild(textDiv);
```